src/build.mk

```diff
@@ -16,3 +15,8 @@ CPPSRC += $(patsubst $(APPDIR)/%,%,$(foreach %,$(INCLUDE_DIRS_LIB),$(wildcard $(
 # Include all directories in /src/
 INCLUDE_DIRS_SRC += $(sort $(dir $(wildcard $(APPDIR)/src/*/)))
 INCLUDE_DIRS += $(INCLUDE_DIRS_SRC)
+
+# Define constants to remove compilation warnings from external libs
+EXTRA_CFLAGS += '-DARDUINO=0'
+EXTRA_CFLAGS += '-DDEBUG_RXANY=0'
```

Comment:
Would you be able to also add a comment regarding which libraries these flags are for? It'd be useful to track since these flags are applied across code globally, so it'd be best to document why we're doing this.